charms/kserve-controller/src/default-custom-images.json

```diff
@@ -10,7 +10,7 @@
     "serving_runtimes__kserve_mlserver": "docker.io/seldonio/mlserver:1.3.2",
     "serving_runtimes__paddleserver": "kserve/paddleserver:v0.11.1",
     "serving_runtimes__pmmlserver": "kserve/pmmlserver:v0.11.1",
-    "serving_runtimes__sklearnserver": "kserve/sklearnserver:v0.11.1",
+    "serving_runtimes__sklearnserver": "charmedkubeflow/sklearnserver:0.11.0-09a6562",
```

Comment:
I did not notice that in the [rock PR](https://github.com/canonical/kserve-rocks/pull/23) we were using an older version of this image. I don't see any changes between 0.11.0 and 0.11.1 Dockerfiles, but for correctness we should update the rock and point this image to `0.11.1-<hash>`.

Reply:
Updated this to 0.11.2

Reply:
```suggestion
    "serving_runtimes__sklearnserver": "charmedkubeflow/sklearnserver:0.11.2-e54c69e",
```